api/src/test/java/org/openmrs/module/edtriageapp/api/EdTriageAppServiceTest.java

```diff
@@ -13,67 +13,68 @@
  */
 package org.openmrs.module.edtriageapp.api;
 
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertNotNull;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.openmrs.Encounter;
 import org.openmrs.api.context.Context;
-import org.openmrs.test.BaseModuleContextSensitiveTest;
-import org.openmrs.test.Verifies;
+import org.openmrs.test.jupiter.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
 
+import java.sql.SQLException;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
 /**
- * Tests {@link ${EdTriageAppService}}.
+ * Tests {@link EdTriageAppService}.
  */
 public class EdTriageAppServiceTest extends BaseModuleContextSensitiveTest {
+
     private static final Log log = LogFactory.getLog(EdTriageAppServiceTest.class);
-    @Test
-    public void shouldSetupContext() {
-        assertNotNull(Context.getService(EdTriageAppService.class));
-    }
 
-    private EdTriageAppService service;
+    @Autowired
+    EdTriageAppService edTriageAppService;
```

Comment:
Can we Autowired services in 2.5.x without the slowndown issues?  Woot if so!

Reply:
My recollection is that we can and have always been able to autowire services into other beans, especially tests.  The slowness issue was caused when we use the autowire annotation inside service implementation classes.

Reply:
Oh, duh, right!